src/agent/reqwest-retry/src/lib.rs

```diff
@@ -107,7 +115,7 @@ pub async fn send_retry_reqwest<
                 max_elapsed_time: Some(max_elapsed_time),
                 ..ExponentialBackoff::default()
             },
-            |err, _| println!("Transient error: {}", err),
+            |err, _| warn!("transient http error: {}", err),
```

Comment:
can we add the url in the message here it might be helpful when debugging.

Reply:
Unfortunately, the URL here isn't easily accessible.  At this context, we have a Builder, which we'd have to TryClone to get access into the URL.

I agree that it would be valuable.  Perhaps a follow-up can refactor this to make it accessible?